mscore/exportxml.cpp

```diff
@@ -5131,6 +5131,8 @@ void ExportMusicXml::write(QIODevice* dev)
                                     xml.tag("staff-distance", QString("%1").arg(QString::number(staffDist,'f',2)));
                                     xml.etag();
                                     }
+                              }
+                        }
 #endif //MERGE
```

Comment:
These unbalanced braces, regardless the code section being disabled, causes a warning from lupdate.